### PR TITLE
Adding details about the actual variant on PUT request for update

### PR DESCRIPTION
**Pull request by matzew**
@wei-lee @sebastienblanc please review

### CODE REVIEW

**Comment by lfryc** [comment]
code-wise seems good, all 5 supported platforms covered

File: jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ public Response updateWindowsVariation(
             windowsVariant.setName(updatedWindowsVariant.getName());
             windowsVariant.setDescription(updatedWindowsVariant.getDescription());
             variantService.updateVariant(windowsVariant);
-            return Response.noContent().build();
+            return Response.ok(windowsVariant).build();
```
Comment (sebastienblanc) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java]: I wonder if must not retrieve the correct subtype like we do in the `GET` , https://github.com/matzew/aerogear-unifiedpush-server/blob/ReturnVariantDetailsOnUpdate/jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java#L96

Comment (lfryc) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java]: There we list all Windows variants, while here the variant is known.

**Comment by lfryc** [comment]
The build fails for unrelated reason, could you someone hit Rebuild button?

File: jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java (hunk 1 above)
Comment (sebastienblanc) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java]: Ah right :) , I must wake up :)

Comment (matzew) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java]: is the update (one) variant endpoint :)

**Comment by matzew** [comment]
@lfryc it crashes in the migrator - I have no clue :) 

totally unrelated

**Comment by wei-lee** [comment]
@matzew Thanks for the update!

**Pull request merged**